### PR TITLE
Add option to ignore filebeat registry file check

### PR DESCRIPTION
**Pull request by OpeOnikute**
### What does this PR do?

- Adds an option to ignore the Filebeat registry file check.
- Releases a new version (1.1.0) with this change.

### Motivation

What inspired you to submit this pull request?
- Filebeat changed the registry file format from version 7, blocking people using that version from monitoring with this integration. See details in [this issue](https://github.com/DataDog/integrations-extras/issues/915).
- This integration does two things: A registry file check and a service check using the HTTP endpoint. Instead of remaining blocked by the implementation of a new structure for the registry file, it would be prudent to allow users that need only the HTTP check use this integration by turning off the registry check with a configuration parameter.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

I looked through the tests and the convention seems to be no tests for whether a config option is working properly, so I didn't add one. 

EDIT: Added one test that the config works

### CODE REVIEW

**Comment by hithwen** [comment]
Pinging @wk8 as integration maintainer

**Comment by hithwen** [comment]
LGTM but a test would be nice

**Comment by OpeOnikute** [comment]
@hithwen thanks, added a test for the new config option

**Comment by OpeOnikute** [comment]
@wk8 doesn't seem to be active on this of recent. 

@coignetp @npezzotti I noticed you made the most recent changes. Please can you help review this?

File: filebeat/tests/test_filebeat.py [hunk 1]
```diff
@@ -101,6 +106,18 @@ def test_missing_registry_file(aggregator):
...
 
 
+@pytest.mark.usefixtures('dd_environment')
+def test_ignore_registry(aggregator, instance):
+    instance['registry_file_path'] = "malformed_json"
+    instance["ignore_registry"] = True
+    check = FilebeatCheck("filebeat", {}, [instance])
+    # test that it silently ignores the registry file
+    # and does the http check
+    check.check(instance)
```
Comment (hithwen) [inline filebeat/tests/test_filebeat.py:116]:
```suggestion
    check.check(instance)
    aggregator.assert_metric('registry.unprocessed_bytes', count=0)
```

**Review by hithwen — APPROVED** [review]

**Pull request merged**